SMLHelper.sln

```diff
@@ -11,22 +11,22 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SMLHelper.Tests", "SMLHelpe
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
+		BELOWZERO|Any CPU = BELOWZERO|Any CPU
+		SUBNAUTICA|Any CPU = SUBNAUTICA|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{418502DD-372D-4EF9-8021-B262552DFEDE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{418502DD-372D-4EF9-8021-B262552DFEDE}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{418502DD-372D-4EF9-8021-B262552DFEDE}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{418502DD-372D-4EF9-8021-B262552DFEDE}.Release|Any CPU.Build.0 = Release|Any CPU
-		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.Release|Any CPU.Build.0 = Release|Any CPU
+		{418502DD-372D-4EF9-8021-B262552DFEDE}.BELOWZERO|Any CPU.ActiveCfg = BELOWZERO|Any CPU
+		{418502DD-372D-4EF9-8021-B262552DFEDE}.BELOWZERO|Any CPU.Build.0 = BELOWZERO|Any CPU
+		{418502DD-372D-4EF9-8021-B262552DFEDE}.SUBNAUTICA|Any CPU.ActiveCfg = SUBNAUTICA|Any CPU
+		{418502DD-372D-4EF9-8021-B262552DFEDE}.SUBNAUTICA|Any CPU.Build.0 = SUBNAUTICA|Any CPU
+		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.BELOWZERO|Any CPU.ActiveCfg = BELOWZERO|Any CPU
+		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.BELOWZERO|Any CPU.Build.0 = BELOWZERO|Any CPU
+		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.SUBNAUTICA|Any CPU.ActiveCfg = SUBNAUTICA|Any CPU
+		{C8FB0981-77D2-47C7-BBEF-A3A9EBACACBF}.SUBNAUTICA|Any CPU.Build.0 = SUBNAUTICA|Any CPU
+		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.BELOWZERO|Any CPU.ActiveCfg = BELOWZERO|Any CPU
+		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.BELOWZERO|Any CPU.Build.0 = BELOWZERO|Any CPU
+		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.SUBNAUTICA|Any CPU.ActiveCfg = SUBNAUTICA|Any CPU
+		{EBC4AAF3-A13B-427E-BF99-3AE23EDEB4B8}.SUBNAUTICA|Any CPU.Build.0 = SUBNAUTICA|Any CPU
```

Comment:
Something like this will probably need to be done in the QModManager solution as well when the merge happens